src/documents/ContextAwareness.md

```diff
@@ -0,0 +1,327 @@
+---
+title: ContextAwareness API
+layout: default
+category: API
+---
+
+The ContextAwareness API of Infusion provides a powerful suite of grades and utilities that allow a component to be responsive to aspects of its contexts in a flexible and open-ended way.
+For lightweight cases of context adaptation, the [`distributeOptions`](IoCSS.md) scheme suffices, in that it enables a component to be responsive to broadcasts "advising" it from around the
```

Comment:
I'm not convinced that "broadcast" is the right word here (and in the rest of this page). It's possible that I'm not actually understanding, but "broadcast" implies sending information out, for anyone to hear, not knowing or caring who listens or acts on the information – more like events. But if I'm understanding this API correctly (which I may not be!), this sounds much more targeted and deliberate than that, in which case I would encourage you to come up with better terminology than "broadcast".